src/modules/Search/Search.js

```diff
@@ -166,6 +166,11 @@ export default class Search extends Component {
     /** A search can have its results take up the width of its container. */
     fluid: PropTypes.bool,
 
+    /** A search input can take up the width of its container . */
+    input: PropTypes.shape({
```

Comment:
Alright, just merged the update.  You can now update this PR with the latest master and you'll have the `Input.create()` method 👍